### ISSUE TITLE
restore downward volume failed

### ISSUE THREAD

**Issue by stoneshi-yunify** [jibutech/backup-saas-client#37]
```
$ velero --kubeconfig=/tmp/config.yy -n qiming-backend restore logs c01-c02-vpc-cfh9o-d8kz2  | grep "level=error"
time="2021-11-03T06:05:02Z" level=error msg="unable to successfully complete restic restores of pod's volumes" error="pod volume restore failed: error restoring volume: error running restic restore, cmd=restic restore --repo=s3:https://s3.kubesphere.cloud/online-bucket-340556865551237290-spf0l/379674222693220542/restic/cloud-test --password-file=/tmp/credentials/qiming-backend/velero-restic-credentials-repository-password --cache-dir=/scratch/.cache/restic 903e7cae --target=., stdout=restoring <Snapshot 903e7cae of [/host_pods/1d3866fd-f049-4220-b401-0467bb8fed08/volumes/kubernetes.io~downward-api/istio-podinfo] at 2021-11-03 03:58:04.534199819 +0000 UTC by root@velero> to .\n, stderr=ignoring error for /..data: Symlink: symlink ..2021_11_03_02_05_36.028999435 /host_pods/1b40e18f-1c17-40fb-a084-5e46f7c36dd2/volumes/kubernetes.io~downward-api/istio-podinfo/..data: file exists\nignoring error for /annotations: Symlink: symlink ..data/annotations /host_pods/1b40e18f-1c17-40fb-a084-5e46f7c36dd2/volumes/kubernetes.io~downward-api/istio-podinfo/annotations: file exists\nignoring error for /labels: Symlink: symlink ..data/labels /host_pods/1b40e18f-1c17-40fb-a084-5e46f7c36dd2/volumes/kubernetes.io~downward-api/istio-podinfo/labels: file exists\nFatal: There were 3 errors\n\n: exit status 1" logSource="pkg/restore/restore.go:1438" restore=qiming-backend/c01-c02-vpc-cfh9o-d8kz2
time="2021-11-03T06:05:08Z" level=error msg="unable to successfully complete restic restores of pod's volumes" error="pod volume restore failed: error restoring volume: error running restic restore, cmd=restic restore --repo=s3:https://s3.kubesphere.cloud/online-bucket-340556865551237290-spf0l/379674222693220542/restic/cloud-test --password-file=/tmp/credentials/qiming-backend/velero-restic-credentials-repository-password --cache-dir=/scratch/.cache/restic 39f30355 --target=., stdout=restoring <Snapshot 39f30355 of [/host_pods/def69738-2f21-47fc-9691-9697261c1528/volumes/kubernetes.io~downward-api/istio-podinfo] at 2021-11-03 03:57:56.099126157 +0000 UTC by root@velero> to .\n, stderr=ignoring error for /..data: Symlink: symlink ..2021_11_03_02_05_36.204042990 /host_pods/257cd664-201d-4b73-a19e-3ea04d10e577/volumes/kubernetes.io~downward-api/istio-podinfo/..data: file exists\nignoring error for /annotations: Symlink: symlink ..data/annotations /host_pods/257cd664-201d-4b73-a19e-3ea04d10e577/volumes/kubernetes.io~downward-api/istio-podinfo/annotations: file exists\nignoring error for /labels: Symlink: symlink ..data/labels /host_pods/257cd664-201d-4b73-a19e-3ea04d10e577/volumes/kubernetes.io~downward-api/istio-podinfo/labels: file exists\nFatal: There were 3 errors\n\n: exit status 1" logSource="pkg/restore/restore.go:1438" restore=qiming-backend/c01-c02-vpc-cfh9o-d8kz2
time="2021-11-03T06:05:15Z" level=error msg="unable to successfully complete restic restores of pod's volumes" error="pod volume restore failed: error restoring volume: error running restic restore, cmd=restic restore --repo=s3:https://s3.kubesphere.cloud/online-bucket-340556865551237290-spf0l/379674222693220542/restic/cloud-test --password-file=/tmp/credentials/qiming-backend/velero-restic-credentials-repository-password --cache-dir=/scratch/.cache/restic 579aeee2 --target=., stdout=restoring <Snapshot 579aeee2 of [/host_pods/8bb024f8-ea47-49d5-8f9f-be982ad464c2/volumes/kubernetes.io~downward-api/istio-podinfo] at 2021-11-03 03:58:22.037961419 +0000 UTC by root@velero> to .\n, stderr=ignoring error for /..data: Symlink: symlink ..2021_11_03_02_05_36.919395306 /host_pods/8da6fc16-f3f9-4541-94fd-b617ff2734c8/volumes/kubernetes.io~downward-api/istio-podinfo/..data: file exists\nignoring error for /annotations: Symlink: symlink ..data/annotations /host_pods/8da6fc16-f3f9-4541-94fd-b617ff2734c8/volumes/kubernetes.io~downward-api/istio-podinfo/annotations: file exists\nignoring error for /labels: Symlink: symlink ..data/labels /host_pods/8da6fc16-f3f9-4541-94fd-b617ff2734c8/volumes/kubernetes.io~downward-api/istio-podinfo/labels: file exists\nFatal: There were 3 errors\n\n: exit status 1" logSource="pkg/restore/restore.go:1438" restore=qiming-backend/c01-c02-vpc-cfh9o-d8kz2
time="2021-11-03T06:05:21Z" level=error msg="unable to successfully complete restic restores of pod's volumes" error="pod volume restore failed: error restoring volume: error running restic restore, cmd=restic restore --repo=s3:https://s3.kubesphere.cloud/online-bucket-340556865551237290-spf0l/379674222693220542/restic/cloud-test --password-file=/tmp/credentials/qiming-backend/velero-restic-credentials-repository-password --cache-dir=/scratch/.cache/restic 2cf3bb93 --target=., stdout=restoring <Snapshot 2cf3bb93 of [/host_pods/02ce3ee0-e2b8-45ac-9971-ca01a7940c64/volumes/kubernetes.io~downward-api/istio-podinfo] at 2021-11-03 03:58:30.805727737 +0000 UTC by root@velero> to .\n, stderr=ignoring error for /..data: Symlink: symlink ..2021_11_03_02_50_34.110709527 /host_pods/7ae83c17-6d91-4f94-adb4-3cc40bd52010/volumes/kubernetes.io~downward-api/istio-podinfo/..data: file exists\nignoring error for /annotations: Symlink: symlink ..data/annotations /host_pods/7ae83c17-6d91-4f94-adb4-3cc40bd52010/volumes/kubernetes.io~downward-api/istio-podinfo/annotations: file exists\nignoring error for /labels: Symlink: symlink ..data/labels /host_pods/7ae83c17-6d91-4f94-adb4-3cc40bd52010/volumes/kubernetes.io~downward-api/istio-podinfo/labels: file exists\nFatal: There were 3 errors\n\n: exit status 1" logSource="pkg/restore/restore.go:1438" restore=qiming-backend/c01-c02-vpc-cfh9o-d8kz2
time="2021-11-03T06:05:28Z" level=error msg="unable to successfully complete restic restores of pod's volumes" error="pod volume restore failed: error restoring volume: error running restic restore, cmd=restic restore --repo=s3:https://s3.kubesphere.cloud/online-bucket-340556865551237290-spf0l/379674222693220542/restic/cloud-test --password-file=/tmp/credentials/qiming-backend/velero-restic-credentials-repository-password --cache-dir=/scratch/.cache/restic 3f9639a8 --target=., stdout=restoring <Snapshot 3f9639a8 of [/host_pods/4c88b990-00ab-458d-ac95-01f03f27d958/volumes/kubernetes.io~downward-api/istio-podinfo] at 2021-11-03 03:58:13.2325933 +0000 UTC by root@velero> to .\n, stderr=ignoring error for /..data: Symlink: symlink ..2021_11_03_02_05_36.751502709 /host_pods/45647b72-7377-45bd-994f-588e1384707e/volumes/kubernetes.io~downward-api/istio-podinfo/..data: file exists\nignoring error for /annotations: Symlink: symlink ..data/annotations /host_pods/45647b72-7377-45bd-994f-588e1384707e/volumes/kubernetes.io~downward-api/istio-podinfo/annotations: file exists\nignoring error for /labels: Symlink: symlink ..data/labels /host_pods/45647b72-7377-45bd-994f-588e1384707e/volumes/kubernetes.io~downward-api/istio-podinfo/labels: file exists\nFatal: There were 3 errors\n\n: exit status 1" logSource="pkg/restore/restore.go:1438" restore=qiming-backend/c01-c02-vpc-cfh9o-d8kz2
```

**Comment by jerry-jibu** [comment]
fix is in https://github.com/vmware-tanzu/velero/pull/4076

**Comment by jerry-jibu** [comment]
will upgrade velero to v1.7.0 for the fix.

**Comment by stoneshi-yunify** [comment]
fixed with velero v1.7
